packages/ovee/src/utils/isString.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Comes from `lodash` `isString` method
```

Comment:
Add a link to Lodash licensing to make it clear

Reply:
sure